packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -150,7 +181,21 @@ export const DateInput: React.VFC<DateInputProps> = ({
         onFocus={handleFocus}
         {...inputProps}
       />
-
+      {status && (
+        <div
+          className={classnames(styles.message, {
+            [styles.disabled]: disabled,
+          })}
+        >
```

Comment:
Interesting. Is this an existing pattern we have? I can't imagine us showing a validation message if the field is disabled (but correct me if I'm wrong).